pkg/plugin/validator_test.go

```diff
@@ -107,6 +114,18 @@ func TestMatchIssue(t *testing.T) {
 			wantErrSubstr:           "issue is in state: closed",
 			issueBytes:              []byte(`{"state": "closed"}`),
 		},
+		{
+			name:     "issue_not_exist",
+			issueURL: fmt.Sprintf("%s%s", issueURLHost, noneExistIssueURLPath),
+			cfg: &PluginConfig{
+				GitHubAppID:             "test-github-id",
+				GitHubAppInstallationID: "test-install-id",
+				GitHubAppPrivateKeyPEM:  testPrivateKeyString,
+			},
+			fakeTokenServerResqCode: http.StatusCreated,
+			wantErrSubstr:           "issue not found",
+			issueBytes:              []byte(`{"state": "closed"}`),
```

Comment:
We should verify what type of error in test cases too.

Reply:
Added